config/initializers/contact_us.rb

```diff
@@ -27,4 +27,4 @@
   # Set the following variable to true if you are using localized paths
   # e.g. /en/contact-us OR /fr/contact-us
   config.localize_routes = true
```

Comment:
We are not using that anymore, and if so it would be query param for the language...